### DISCUSSION TITLE
Slides / Presentation Size and Template

### DISCUSSION THREAD

**Discussion by sirinath** [typst/typst#215] [Ideas]
Can we have
1. paper size for a presentation
2. couple of presentation templates
please.

**Comment by laurmaedje** [comment]
There are already paper sizes for presentation (and you can always set a custom width/height). Regarding templates: We want to support presentations down the road, but can't focus on them right now, so templates would need to come from the community for now.

**Reply by sirinath** [reply]
Thanks  for the update.
